src/old/lib/utilities/API/api.ts

```diff
@@ -306,6 +308,16 @@ export const getPlatformInformation = async (
   return instance.get(`/platform-information/${id}`);
 };
 
+export const fetchMaterialsChange = async (
```

Comment:
I propose to name on `fetchChangeLog` or in some other way but to make `changeLog` to be part of the name